extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -804,7 +808,10 @@ describes.repeated('', {
         expect(form.className).to.not.contain('amp-form-submit-error');
         expect(form.className).to.not.contain('amp-form-submit-success');
         fetchRejecter();
-        return timer.promise(5).then(() => {
+
+        return ampForm.xhrSubmitPromiseForTesting().then(() => {
+          expect(false).to.be.true; // should not run
```

Comment:
Throwing a plain error is concise, but `assert` seems more idiomatic for Chai. Any objections to using `assert`?